src/store/actions/groups.js

```diff
@@ -0,0 +1,19 @@
+import actionTypes from '../actionTypes'
+import { frApiRequest } from './services'
+
+export const getGroups = (params) => {
```

Comment:
There shouldn't really be a need to get groups since they're always included with a user, but sure we can add these for completeness.

Also the action type for `getGroups` should be `groups.search` and not `groups.read`. `read` is for a single resource and `search` is for listing endpoints. This mirrors terminology used on the API.